### PR TITLE
Al listings

### PR DESCRIPTION
**Pull request by pixapi**
This PR adds amenities to listings table, improves tests user sees a listing, fixes listings index paragraph ellipsis.

### CODE REVIEW

**Comment by pixapi** [comment]
All 36 are passing. Please review and merge if ok.

File: db/migrate/20170330005638_add_number_baths_to_listings.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    add_column :listings, :dryer, :boolean, default: false
+    add_column :listings, :hair_dryer, :boolean, default: false
+    add_column :listings, :washer, :boolean, default: false
+    add_column :listings, :tv, :boolean, default: false
+    add_column :listings, :buzzer_wireless_intercom, :boolean, default: false
+    add_column :listings, :iron, :boolean, default: false
+    add_column :listings, :essentials, :boolean, default: false
+    add_column :listings, :laptop_friendly_workspace, :boolean, default: false
+    add_column :listings, :heating, :boolean, default: false
+    add_column :listings, :private_entrance, :boolean, default: false
```
Comment (meyerhoferc) [inline db/migrate/20170330005638_add_number_baths_to_listings.rb:31]: great additions here

File: spec/features/user_can_see_a_listing_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    click_on "#{listing_1.title}"
+
+    expect(current_path).to eq listing_path(listing_1)
+
+    expect(page).to have_content(listing_1.title)
+    expect(page).to have_content(listing_1.description)
+    expect(page).to have_xpath("//img[@src='#{listing_1.images[0].image_url}']")
+    expect(page).to have_xpath("//img[@src='#{listing_1.images[1].image_url}']")
+    expect(page).to have_xpath("//img[@src='#{listing_1.images[2].image_url}']")
+    expect(page).to have_xpath("//img[@src='#{listing_1.user.user_photo}']")
```
Comment (meyerhoferc) [inline spec/features/user_can_see_a_listing_spec.rb:19]: 👍

**Review by meyerhoferc — APPROVED** [review]
looks great, merging now

**Pull request merged**